Documentation/guides/building-apps/build-properties.md

```diff
@@ -832,6 +832,18 @@ APK root directory. The format of the path is `lib\ARCH\wrap.sh` where
 + `x86_64`
 + `x86`
 
+## AndroidIncludeAssetPacksInPackage
+
+This property controls if an Asset Packs build automatically are auto
+included in the final `.aab` file. It will default to `true`.
```

Comment:
Which, to my earlier question, implies that (by default) the `.aab` *will* contain asset packs, and they *won't* be separate files.  Good to know!